cluster/examples/kubernetes/ceph/csi/rbac/cephfs/csi-provisioner-rbac.yaml

```diff
@@ -8,6 +8,18 @@ kind: ClusterRole
 apiVersion: rbac.authorization.k8s.io/v1
 metadata:
   name: cephfs-external-provisioner-runner
+aggregationRule:
```

Comment:
@Madhu-1 Should we update the CSI roles in a separate PR so they can be updated in the ceph-csi repo first? Or shall we go ahead with this change and then port them back to ceph-csi? Either way is fine with me

Reply:
@travisn same works for me, @kshlm can you send a PR for the ceph-csi driver with these changes?